packages/avatar-creator/src/scripts/avatar-loader.ts

```diff
@@ -17,6 +17,7 @@ import {
 } from "playcanvas";
 import type { GlbContainerResource } from "playcanvas/build/playcanvas/src/framework/parsers/glb-container-resource";
 
+import animGraphData from "../assets/anim-graph.json";
```

Comment:
This JSON file creates some indirection and lack of typing and it seems that there are quite significant additions made to the structure at runtime. Can we just build the structure within this code rather than import JSON?

Reply:
It gets modified as a data structure later on in this file. If it were a binary blob that would make sense, but we depend on its structure for that code to run.

```
[...]
      const layer = animGraphData.layers[0];
      const name = item[0];

      // state
      layer.states.push({
        name,
        speed: 1,
        loop: false,
        defaultState: false,
      });
[...]
```